src/v4/enum/enumUIKitID.ts

```diff
@@ -51,7 +51,7 @@ export enum ElementID {
   illustration = 'illustration',
   title = 'title',
   description = 'description',
-  explore_communities_button = 'explore_communities_button',
+  explore_communities_button = 'explore_communitties_button',
```

Comment:
```suggestion
  explore_communities_button = 'explore_communities_button',
```

Reply:
That wrong type string `explore_communitties_button` is from config json in tech spec. I am not sure should I correct spelling or not. That's why I use the wrong typo xD

Should I change ?

Reply:
Updated xD

Reply:
> That wrong type string `explore_communitties_button` is from config json in tech spec. I am not sure should I correct spelling or not. That's why I use the wrong typo xD
> 
> Should I change ?

I like to say YES, and please ping others about this thing as well

Reply:
Yes. already changed and will let them know